customer-wallet/src/containers/MainLayout/index.js

```diff
@@ -14,13 +14,28 @@ import CircularProgress from 'material-ui/CircularProgress'
 import SwitchUserMenu from './SwitchUserMenu'
 // data
 import usersType from 'shared-libraries/lib/fixtures/usersType'
+// styles
+import { mainColor } from 'context/palette'
+// assets
+import chluLogo from 'images/svg/chlu-2.svg'
 // constants
 const { dataActions: { profile: { changeUserType } } } = actions
 
-const style = {
+const CircularProgressStyle = {
```

Comment:
For example, even a small style like this, lets put it into CSS. My reason is that, slowly styles start to grow and it gets confusing to find where the styles are.